### PR TITLE
allPagesModified

### PR DESCRIPTION
**Pull request by anuanushas**
allPagesModified

### CODE REVIEW

File: src/Component/UnderConstruction.jsx [hunk 1]
```diff
@@ -1,9 +1,13 @@
-import React from "react";
+import React, { useEffect } from "react";
 import { Container, Box, Typography, Grid } from "@mui/material";
 import ConstructionIcon from "@mui/icons-material/Construction";
 import CopyRight from "./CopyRight/CopyRight";
+// updated page title
```
[the remark this answers is not in the document]
Comment (anuanushas) [reply src/Component/UnderConstruction.jsx:5]: done

File: src/Pages/Discuss/Discuss.jsx [hunk 2]
```diff
@@ -20,169 +20,183 @@ import NewPost from "./NewPost";
...
-    const [searchTerm, setSearchTerm] = useState("");
-    const [openDialog, setOpenDialog] = useState(false);
-    const [sortBy, setSortBy] = useState("createdAt");
-    const [order, setOrder] = useState("desc");
-    const [page, setPage] = useState(1);
-    const [limit, setLimit] = useState(10);
-    const [totalPages, setTotalPages] = useState(0);
+  React.useEffect(() => {
+    document.title = "CodeRoutine | Discuss";
+  }, []);
```
Comment (aakash416) [inline src/Pages/Discuss/Discuss.jsx:25]: after state add

File: src/Pages/AddProblem/AddProblem.jsx [hunk 3]
```diff
@@ -17,6 +17,9 @@ import { addProblem } from "../../Api/problemApi";
 const difficultyOptions = ["Easy", "Medium", "Hard"];
 
 const AddProblem = () => {
+  React.useEffect(() => {
```
[the remark this answers is not in the document]
Comment (anuanushas) [reply src/Pages/AddProblem/AddProblem.jsx:20]: done

File: src/Pages/Discuss/DiscussDetails.jsx [hunk 4]
```diff
@@ -21,10 +21,17 @@ import { ContextStore } from "../../Context/ContextStore";
...
+  addLikeOrRemoveLike,
+  deleteDiscussById,
+  getDiscussById,
+} from "../../Api/Discuss/discussApi";
 import { addCommentToTopic } from "../../Api/Discuss/commentApi";
 
 const DiscussDetails = () => {
+  React.useEffect(() => {
+    document.title = "CodeRoutine | Discuss Details";
+  }, []);
```
Comment (aakash416) [inline src/Pages/Discuss/DiscussDetails.jsx:34]: after state add

File: src/Pages/Interview/InterviewOnline.jsx [hunk 5]
```diff
@@ -15,6 +14,9 @@ import {
 } from "@mui/material";
 
 function InterviewOnline() {
+  React.useEffect(() => {
+    document.title = "CodeRoutine | Interview Online";
+  }, []);
```
Comment (aakash416) [inline src/Pages/Interview/InterviewOnline.jsx:19]: after state add do

File: src/Pages/LoginSignUp/SignUp.jsx [hunk 6]
```diff
@@ -62,6 +62,9 @@ const SkeletonLoader = () => (
 );
 
 export default function SignUp() {
+  React.useEffect(() => {
+    document.title = "CodeRoutine | Sign Up";
+  }, []);
```
Comment (aakash416) [inline src/Pages/LoginSignUp/SignUp.jsx:67]: after state add do

File: src/Pages/Courses/CreateCourse.jsx [hunk 7]
```diff
@@ -14,6 +14,9 @@ import TopicForm from "./TopicForm";
 import { useNavigate } from "react-router-dom";
 
 const CreateCourse = () => {
+  React.useEffect(() => {
+    document.title = "CodeRoutine | Create Course";
+  }, []);
```
[the remark this answers is not in the document]
Comment (anuanushas) [reply src/Pages/Courses/CreateCourse.jsx:19]: done

**Review by anuanushas — COMMENTED** [review]
done 3 tasks

File: src/Pages/Discuss/Discuss.jsx (hunk 2 above)
Comment (anuanushas) [reply src/Pages/Discuss/Discuss.jsx:25]: done

File: src/Pages/Discuss/DiscussDetails.jsx (hunk 4 above)
Comment (anuanushas) [reply src/Pages/Discuss/DiscussDetails.jsx:34]: done

File: src/Pages/Interview/InterviewOnline.jsx (hunk 5 above)
Comment (anuanushas) [reply src/Pages/Interview/InterviewOnline.jsx:19]: done

File: src/Pages/LoginSignUp/SignUp.jsx (hunk 6 above)
Comment (anuanushas) [reply src/Pages/LoginSignUp/SignUp.jsx:67]: done

**Review by anuanushas — COMMENTED** [review]
doneall  tasks

**Pull request closed without merging**